x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/TemplateParamValidator.java

```diff
@@ -90,7 +91,7 @@ public void validate(Map<String, Object> templateParams) throws ValidationExcept
             secondParam = AccessController.doPrivileged((PrivilegedExceptionAction<JsonNode>) () -> {
                 return OBJECT_MAPPER.valueToTree(templateParams);
             });
-        } catch (Exception e) {
+        } catch (PrivilegedActionException e) {
             throw new ElasticsearchException("failed to convert parameters while validating", e);
```

Comment:
Do we have a test that verifies that this returns a 400 error to the user? If not, I think we should add it. Sorry for the extra iteration...

Reply:
@Mikep86 I couldn't find a good way to make the function to throw this exception therefore check if we are getting a 500. 

One way to test this is extracting the `try/catch` and pass something that I can mock there to throw which looks awkward to me as only thing I can really pass is `OBJECT_MAPPER` that is defined as final

Any suggestions?

Reply:
Ok, we're probably fine without a test then. Can you change this to throw `ElasticsearchStatusException`? That way we can force a 500 response.

Reply:
My opinion: Since the fix here was updating our security policy, a happy path check that we can process floats in our yaml tests should suffice here.

Reply:
++ to what Kathleen said, plus changing this to throw `ElasticsearchStatusException`

Reply:
Perfect! Thank you both <3